TableSDK/Table SDK/SDK/ViewControllers/JitsiVideo/JitsiVideoVC.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  JitsiVideoVC.swift
+//  Table SDK
+//
+//  Created by Afify on 04/11/2020.
+//  Copyright © 2020 TableCO. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import JitsiMeet
+
+class JitsiVideoVC: UIViewController  {
+    @IBOutlet var jitsiMeetView: JitsiMeetView!
+    var tenant = ""
+    var roomID = ""
+    var token = ""
+    var userInfo: UserAttributes = UserAttributes()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.navigationController?.navigationBar.isHidden = true
+        let tokenizedURL = "https://8x8.vc/" + self.tenant + "/" + self.roomID + "?jwt=" + self.token
```

Comment:
Is this URL going to be the same with dev / production? Do we need to double check with the Table team?